src/libraries/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -283,6 +289,9 @@
     <Reference Include="System.Xml.ReaderWriter" />
     <Reference Include="System.Xml.XmlSerializer" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Permissions\src\System.Security.Permissions.csproj" />
```

Comment:
Why is this ProjectReference added?

Reply:
added the comment. for more info https://github.com/dotnet/runtime/issues/52982

Reply:
Cool. I didn't see that it already existed for .NETStandard below and was just moved. Thanks for clarifying and adding a comment.